features/wishlist.feature

```diff
@@ -0,0 +1,20 @@
+Feature: The wishlist service back-end
+    As a e-Commerce Website
+    I need a RESTful wishlist service
+    So that I can keep track of all users' wishlists and their contents
+
+Scenario: Create a Wishlist
```

Comment:
I believe we need to work with the BDD format:
`Given` (set of preconditions)
`When` (something happens)
`Then` (testable outcome)

I feel the `Given` is missing here?
Maybe 
```
Given the following wishlists
| id   | name           | customer_id  |
| 1    | WISHLIST       | 10           |

Reply:
We can add it when we need it though

Reply:
Should we add it before any `Then` statement?

Reply:
This BDD is to create a wishlist. We don't need to preload any data

Reply:
Just at the beginning of the feature file. But I mean they are used for listing and updating right. We don't need dummy data for create wishlist.

Reply:
Or maybe: `Given the server is running`
I believe we just need a `Given` statement (requirement from the Gherkin syntax)

Reply:
Yeah. Ok. I'll take a look at this

Reply:
@lombardero Added

Reply:
Looks great now!